### PR TITLE
CASMINST-5202 remove auto retry; use ipmi over ssh to reboot

### PR DESCRIPTION
**Pull request by dzou-hpe**
# Description

- remove auto retry
- use ipmi over ssh to reboot

# Testing
surtur

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by johren-hpe** [comment]
This looks like it is removing for all NCNs and not must master like we discussed in stand-up.   Am I reading that correctly?

File: upgrade/scripts/common/ncn-rebuild-common.sh [hunk 1]
```diff
@@ -246,46 +246,7 @@ state_recorded=$(is_state_recorded "${state_name}" ${target_ncn})
...
-        done
-        # Make sure we found a pod that worked
-        [[ -n ${IPMI_USERNAME} ]]
-
-        # power cycle node
-        ipmitool -I lanplus -U "${IPMI_USERNAME}" -E -H "${target_ncn_mgmt_host}" chassis power off
-        sleep 20
-        ipmitool -I lanplus -U ${IPMI_USERNAME} -E -H $target_ncn_mgmt_host chassis power status
-        ipmitool -I lanplus -U ${IPMI_USERNAME} -E -H $target_ncn_mgmt_host chassis power on
+        ssh "${target_ncn}" 'ipmitool chassis power cycle'
```
Comment (johren-hpe) [inline upgrade/scripts/common/ncn-rebuild-common.sh:249]: This looks like it is removing for all NCNs and not must master like we discussed in stand-up. Am I reading that correctly?

**Review by johren-hpe — CHANGES_REQUESTED** [review]

File: upgrade/scripts/common/ncn-rebuild-common.sh (hunk 1 above)
Comment (dzou-hpe) [reply upgrade/scripts/common/ncn-rebuild-common.sh:249]: yes, this is going to remove it from storage nodes too for now but i think once @leliasen-hpe get storage upgrade moved to argo it is only going to be used by master node. worker nodes are already not using this

Comment (johren-hpe) [reply upgrade/scripts/common/ncn-rebuild-common.sh:249]: Storage node upgrade moved to argo is planned for 1.3 as well?

**Comment by johren-hpe** [comment]
I see that you mentioned in the casm-pet channel that this was tested on surtur.   Please include that in the PR description for future PRs.

**Review by johren-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**